lib/Service/CollectiveShareService.php

```diff
@@ -124,6 +124,12 @@ public function findShare(string $userId, int $collectiveId, int $pageId): ?Coll
 		if ($folderShare->getPassword()) {
 			$collectiveShare->setPassword($folderShare->getPassword());
 		}
+
+		if ($folderShare->getExpirationDate()) {
+			$collectiveShare->setExpirationDate($folderShare->getExpirationDate());
+		} else {
+			$collectiveShare->setExpirationDate(null);
+		}
```

Comment:
Seems like `$folderShare->getExpirationDate()` already returns `null` for unset expiration dates. So I guess it should be save to do `$collectiveShare->setExpirationDate($folderShare->getExpirationDate())` straight away.